terraform/projects/infra-assets/backup.tf

```
@@ -11,44 +11,38 @@ resource "aws_s3_bucket_versioning" "assets_backup" {
   versioning_configuration { status = "Enabled" }
 }
 
-resource "aws_iam_policy" "backup" {
-  count  = var.aws_environment == "production" ? 1 : 0
-  name   = "govuk-${var.aws_environment}-assets-backup-policy"
-  policy = data.template_file.backup_policy[0].rendered
+resource "aws_s3_bucket_policy" "assets_backup" {
+  count    = var.aws_environment == "production" ? 1 : 0
+  provider = aws.backup
+  bucket   = aws_s3_bucket.assets_backup[0].id
+  policy   = data.aws_iam_policy_document.bucket_is_replication_target[0].json
 }
 
-resource "aws_iam_role" "backup" {
+data "aws_iam_policy_document" "bucket_is_replication_target" {
   count = var.aws_environment == "production" ? 1 : 0
-  name  = "govuk-${var.aws_environment}-assets-backup"
-
-  assume_role_policy = <<POLICY
-{
-  "Version": "2012-10-17",
-  "Statement": [
-    {
-      "Action": "sts:AssumeRole",
-      "Principal": { "Service": "s3.amazonaws.com" },
-      "Effect": "Allow",
-      "Sid": ""
+  statement {
+    sid = "S3RoleInSourceAccountReplicatesObjectsToDestinationBucket"
+    principals {
+      type        = "AWS"
+      identifiers = [local.replication_service_role_in_prod_account]
     }
-  ]
-}
-POLICY
-}
-
-resource "aws_iam_policy_attachment" "backup" {
-  count      = var.aws_environment == "production" ? 1 : 0
-  name       = "govuk-${var.aws_environment}-backup-policy-attachment"
-  roles      = [aws_iam_role.backup[0].name]
-  policy_arn = aws_iam_policy.backup[0].arn
-}
-
-data "template_file" "backup_policy" {
-  count    = var.aws_environment == "production" ? 1 : 0
-  template = file("backup_policy.tpl")
-
-  vars = {
-    src_bucket = aws_s3_bucket.assets.id
-    dst_bucket = aws_s3_bucket.assets_backup[0].id
+    actions = [
+      "s3:GetObjectVersionTagging",
+      "s3:Replicate*",
+    ]
+    resources = ["arn:aws:s3:::${aws_s3_bucket.assets_backup[0].id}/*"]
```

Nit: v minor - might be worth referring the arn attribute directly, just for consistency with how we reference other resource types in IAM policies.

```suggestion
    resources = ["${aws_s3_bucket.assets_backup[0].arn}/*"]
```

---

Good catch, thanks! Done.